### DISCUSSION TITLE
Any indicators in the output regarding which turbines are waking which?

### DISCUSSION THREAD

**Discussion by petebachant** [NatLabRockies/floris#508] [Ideas]
I was wondering if there is anything in the output we could use to identify which turbines are waking which other turbines. Is there maybe a line computed under the hood for the wake trajectory that could be used to search for turbines within a certain distance?

**Comment by rafmudaf** [comment]
Hi @petebachant - Unfortunately we don't currently keep any kind of connectivity information between turbines. The wakes are applied on downstream turbines by masks as in [Jensen](https://github.com/NREL/floris/blob/main/floris/simulation/wake_velocity/jensen.py#L146) and [Gauss](https://github.com/NREL/floris/blob/main/floris/simulation/wake_velocity/gauss.py#L170). However, I have been thinking about creating a graph representation of the farm in order to be more selective about how the solver traverses and applies the wake. It sounds like that may also get you the information you need.

**Comment by petebachant** [comment]
Gotcha. So the deflection models don't have some analytic coefficients stored in memory that we could use to reconstruct a line. A graph representation does sound useful! We're just looking for a more exact way to compare to observations in the field.
